packages/@atjson/source-commonmark/src/source.ts

```diff
@@ -1,5 +1,5 @@
 import Document from "@atjson/document";
-import * as MarkdownIt from "markdown-it";
+import MarkdownIt from "markdown-it";
```

Comment:
TBH I don't fully understand why this is necessary, but this was the combination of tsconfig settings and import syntax that got everything working after making this change.

Reply:
isn't this a semantic difference? the latter is equivalent to `import default as MarkdownIt`, right?

Reply:
From what I understand, packages like this that didn't have a default export were throwing an error when using this syntax, but the `esModuleInterop` setting allows this to work.

Reply:
The change "allowSyntheticDefaultImport" altered this.